### PR TITLE
Climber updates and shooter updates post practice

### PR DESCRIPTION
**Pull request by tuffinmuffin**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issue(s)
Issue: <#X or issue-XX > [REMOVE THIS WHEN DONE Describe the issue you're working on here so it ties to the issue and Kanban (IMPORTANT FOR TRACKING). Make sure to create an issue if one doesn't already exist! ]

## Steps to Test
[ Describe how to run your AUTOMATED TESTS and any manual tests that may be need to prove your fix/feature REMOVE THIS WHEN DONE]
1. [First Step]
2. [Second Step]
3. [and so on...]

## Changelog

[ A brief description of the problem statement. REMOVE THIS WHEN DONE ]

## What?

[ Explain the technical implementation of what you're doing. REMOVE THIS WHEN DONE ]

## Why?

[ Explain why you're making this change and what benefits from it. REMOVE THIS WHEN DONE ]

### CODE REVIEW

File: commands/shortyShooter.py [hunk 1]
```diff
@@ -41,7 +43,10 @@ def execute(self):
         else:
             self.shooter.runIntake(0)
 
-        self.shooter.runShooters(self.shooterSpeed())
+        voltage = 0
+        if self.shooterSpeed() > 0.2:
```
Comment (tuffinmuffin) [inline commands/shortyShooter.py:47]:
```suggestion
        # set full speed if enabling shooter
        if self.shooterSpeed() > 0.2:
```

File: subsystem/sparkyShooter.py [hunk 2]
```diff
@@ -12,9 +14,20 @@ def __init__(self) -> None:
...
+        self.leftShootMotor.enableVoltageCompensation(11.5)
+        self.rightShooterMotor.enableVoltageCompensation(11.5)
+        self.leftEncoder = self.leftShootMotor.getEncoder()
+        self.rightEncoder = self.rightShooterMotor.getEncoder()
+
+    def periodic(self) -> None:
+        wpilib.SmartDashboard.putNumber("left shooter speed", self.leftEncoder.getVelocity())
+        wpilib.SmartDashboard.putNumber("right shooter speed", self.rightEncoder.getVelocity())
+        pass
+
```
Comment (tuffinmuffin) [inline subsystem/sparkyShooter.py:26]:
```suggestion
```

File: subsystem/sparkyShooterPivot.py [hunk 3]
```diff
@@ -18,6 +20,11 @@ def __init__(self) -> None:
         #scaled to 0..1 = forward - end limit
         #80:1 use 1/73.38 100:1 use 88.056
         self.encoder.setPositionConversionFactor(1/88.056)
+        self.encoder.setPosition(0)
+
```
Comment (tuffinmuffin) [inline subsystem/sparkyShooterPivot.py:24]:
```suggestion
        
        # Enable softlimit for negative direction. Note that all postions go from 0..~ -1
```

File: subsystem/sparkyShooterPivot.py [hunk 4]
```diff
@@ -71,6 +82,8 @@ def zeroPivot(self, speed : float = 0.2):
             self.zeroed = True
             self.pivotMotor.set(0.0)
             self.encoder.setPosition(0)
+            self.pivotMotor.setSoftLimit(rev.CANSparkMax.SoftLimitDirection.kForward, -0.01)
```
Comment (tuffinmuffin) [inline subsystem/sparkyShooterPivot.py:85]:
```suggestion
            #once zeroed, set softlimit to allow a soft "landing"
            self.pivotMotor.setSoftLimit(rev.CANSparkMax.SoftLimitDirection.kForward, -0.01)
```

File: commands/shortyShooter.py (hunk 1 above)
Comment (ChrisHirsch) [reply commands/shortyShooter.py:48]: Having conversations with yourself is healthy!

**Review by ChrisHirsch — APPROVED** [review]
Yep! Why not??

**Comment by tuffinmuffin** [comment]
Meging with verbal approval from @ChrisHirsch and @raylmorris

**Pull request merged**